src/metabase/models/serialization.clj

```diff
@@ -981,3 +982,21 @@
     (->> (concat vis-column-settings [(mbql-deps viz) link-card-deps])
          (filter some?)
          (reduce set/union))))
+
+(def ^:private cached-vars
+  (filter (comp ::cache meta) (vals (ns-interns *ns*))))
+
+(defn init-cache
```

Comment:
PS, this is public because the macro needs to invoke this function where it is expanded.